flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/http/HttpClientFactory.java

```diff
@@ -21,14 +21,19 @@
 import es.tid.fiware.fiwareconnectors.cygnus.utils.Constants;
 import java.security.KeyManagementException;
 import java.security.NoSuchAlgorithmException;
+import java.security.Principal;
 import java.security.SecureRandom;
 import java.security.cert.X509Certificate;
 import javax.net.ssl.SSLContext;
 import javax.net.ssl.TrustManager;
 import javax.net.ssl.X509TrustManager;
+import org.apache.http.auth.AuthScope;
```

Comment:
Apache HttpComponents has everything necessary to develop Kerberos clients since version 4.2 :)